### ISSUE TITLE
Automatic line wrapping when word count exceeds configured limit

### ISSUE THREAD

**Issue by carun** [ForkIssues/TrackerWin#841]
We have this in vim. This is a nice feature to have.

**Comment by DanPristupov** [comment]
The wordwrap option is on the small toolbar above the diff control
<img width="152" alt="Screen Shot 2020-06-22 at 07 25 17" src="https://user-images.githubusercontent.com/618115/85251392-871af000-b459-11ea-9649-38dba2d69378.png">

**Comment by carun** [comment]
I should've explained a bit more. Automatic word wrap in the commit message body is needed.

**Comment by DanPristupov** [comment]
The word wrap also works there: 
<img width="471" alt="Screen Shot 2020-06-22 at 22 58 49" src="https://user-images.githubusercontent.com/618115/85334687-f1747480-b4db-11ea-8731-4e8db98f3fb7.png">

**Comment by carun** [comment]
It doesn't wrap automatically when we type the commit messages. May be am I missing something?

![image](https://user-images.githubusercontent.com/471374/85338253-c287df80-b496-11ea-90fd-c2a8cae172d9.png)

**Comment by carun** [comment]
@DanPristupov ping.

**Comment by DanPristupov** [comment]
There is an option to wrap paragraph at Ruler in the context menu of the description field:
<img width="558" alt="Screen Shot 2020-09-04 at 09 50 07" src="https://user-images.githubusercontent.com/618115/92213547-06826200-ee94-11ea-8bcb-4559e0cd0f41.png">

**Comment by carun** [comment]
Can this be done automatically without having to use the context menu?

**Comment by DanPristupov** [comment]
No, I think it would be confusing. Do you know any application which behaves this way?

**Comment by carun** [comment]
Most of the apps that developers use does it. I'm not talking about other git apps, but all the editors like VS, vim, emacs, etc does it automatically.

**Comment by carun** [comment]
@DanPristupov the manual wrap feature is broken as well. At times, the lines have two or three words on manual wrap.


![fork-issue-841](https://user-images.githubusercontent.com/471374/97934021-4a290900-1d29-11eb-86ce-7a853d80ebda.gif)

**Comment by carun** [comment]
@DanPristupov wondering if this can be fixed as it is quite annoying to unable to wrap the commits during amend. Should I raise a separate issue for this?

**Comment by carun** [comment]
Hello @DanPristupov touching base to see if there's any update on this.

**Comment by DanPristupov** [comment]
No, still no update.

> all the editors like VS, vim, emacs, etc does it automatically.

How can I do that in Visual Studio?

**Comment by carun** [comment]
> How can I do that in Visual Studio?

Visual Studio git interface is just dumb. VS not supporting word wrap in it's git interface is a missing feature. I seldom use VS git interface when I have Fork already ;)

It is a good practice to have commit messages wrapped at 80 chars. Having Fork do this automatically is nice. We now write our commit messages in Notepad++ and copy paste that into Fork as auto wrap functionality is missing.

> @DanPristupov the manual wrap feature is broken as well. At times, the lines have two or three words on manual wrap.
> 
The manual wrap feature is buggy as it is shown in the screenshot above. I can raise a different issue for that, to not confuse it with the auto wrap functionality.
